server/utils/passport/ensureStudyPermission.js

```diff
@@ -6,9 +6,7 @@ export default async function ensureStudyPermission(req, res, next) {
 
   try {
     const { study } = req.params
-    const { appDb } = req.app.locals
-    const user = await UserModel.findOne(appDb, { uid: req.user })
-    const { blocked, access } = user
+    const { blocked, access } = req.user
```

Comment:
I think we need a user look up here in case the access to the study has changed

Reply:
94abecf